entries/s/storyblocks.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Storyblocks": {
+    "domain": "storyblocks.com",
```

The website doesn't redirect HTTPS requests to @ properly. Better to specify the full URL then
```suggestion
    "domain": "storyblocks.com",
    "url": "https://www.storyblocks.com/",
```